test/testSetStatus.js

```diff
@@ -20,16 +20,33 @@ const logger = Logger.getTestLogger({
 });
 
 describe('Test set-status', () => {
+  const error = 'oh, no!';
+
   it('sets a 500 for an error', () => {
     assert.deepEqual(
-      setStatus({ content: { html: '<html></html>' }, error: 'oh, no!' }, { logger }),
+      setStatus({ content: { html: '<html></html>' }, error }, { logger }),
       {
-        response: {
-          status: 500,
-          body: '',
+        status: 500,
```

Comment:
is this really correct? don't we need the `response` object?

Reply:
you're right.